snuba_sdk/query_visitors.py

```diff
@@ -396,12 +397,23 @@ def _combine(self, query: "main.Query", returns: Mapping[str, None]) -> None:
             raise InvalidQuery("query must have at least one expression in select")
 
         # Non-aggregate expressions must be in the groupby if there is an aggregate
-        non_aggregates = []
+        non_aggregates: List[Union[Column, CurriedFunction, Function]] = []
         has_aggregates = False
+
+        # Legacy queries use aliases as Columns, so we need to compile all the possible
+        # aliases to use in the checks.
+        alias_columns = set()
         for exp in query.select:
             if (
-                not isinstance(exp, (CurriedFunction, Function))
-                or not exp.is_aggregate()
+                isinstance(exp, (CurriedFunction, Function)) and exp.is_aggregate()
+            ) and exp.alias:
```

Comment:
I think this is not properly formatted.
```
   if isinstance(....bla....) and exp.is_aggregate() and exp.alias
```

Reply:
It seems still improperly formatted. Is black doing this ?

Reply:
Sorry forgot to push the change.